data/assets/scene/solarsystem/planets/earth/satellites/misc/hubble.asset

```diff
@@ -0,0 +1,89 @@
+local transforms = asset.require('scene/solarsystem/planets/earth/transforms')
+local satelliteHelper = asset.require('util/tle_helper')
+
+local url = "https://celestrak.com/satcat/tle.php?CATNR=20580"
+local identifier = "HST"
+local filename = "HST.txt"
+local nodes = {}
+local tle = satelliteHelper.downloadTLEFile(asset, url, identifier, filename)
+
+local initializeAndAddNodes = function()
+  local lineElement = satelliteHelper.makeSingleLineElement(tle, filename)
+  local period = satelliteHelper.getPeriodFromElement(lineElement)
+  local path = tle .. "/" .. filename
+
+  local HubblePosition = {
+    Identifier = "HubblePosition",
+    Parent = transforms.EarthInertial.Identifier,
+    Transform = {
+      Translation = {
+        Type = "TLETranslation",
+        Body = identifier,
+        Observer = transforms.EarthInertial.Identifier,
+        File = path,
+        LineNumber = 1
+      },
+      Rotation = {
+        Type = "SpiceRotation",
+        SourceFrame = "GALACTIC",
+        DestinationFrame = "J2000",
+      }
+    },
+    Tag = { "earth_satellite", "hubble" },
+    GUI = {
+      Name = "Hubble Position",
+      Path = "/Solar System/Planets/Earth/Satellites/Hubble",
+      Hidden = true
+    }
+  }
+
+  local HubbleTrail = {
+    Identifier = "HubbleTrail",
+    Parent = transforms.EarthInertial.Identifier,
+    Renderable = {
+      Type = "RenderableTrailOrbit",
+      Translation = {
+        Type = "TLETranslation",
+        Body = identifier,
+        Observer = transforms.EarthInertial.Identifier,
+        File = path,
+        LineNumber = 1
+      },
+      RenderBinMode = "PostDeferredTransparent",
+      Color = { 0.0, 0.902, 0.6},
+      Fade = 1.5,
+      Period = period,
+      Resolution = 320
+    },
+    Tag = { "earth_satellite", "hubble" },
+    GUI = {
+      Name = "Hubble Trail",
+      Path = "/Solar System/Planets/Earth/Satellites/Hubble"
+    }
+  }
+
+  return { HubblePosition, HubbleTrail }
+end
+
+asset.onInitialize(function ()
+  nodes = initializeAndAddNodes()
+  for _, node in ipairs(nodes) do
+    openspace.addSceneGraphNode(node)
+  end
+end)
+
+asset.onDeinitialize(function ()
+  for i=1, #nodes do
+    openspace.removeSceneGraphNode(nodes[#nodes + 1 - i].Identifier)
+  end
+end)
+
+asset.meta = {
+  Name = "Hubble Space Telescope",
```

Comment:
Maybe a bit late to mention it, but for some reason this name made me think that we included the 3D model of the Hubble Space telescope as well 😅 Should we? NASA has a model: https://solarsystem.nasa.gov/resources/2383/hubble-space-telescope-3d-model/ 

Got a little disappointed when I opened the profile and tried to go to look at Hubble